src/smart-components/App/stories/integrated-app.scss

```diff
@@ -0,0 +1,90 @@
+@import '../../../styles/variables';
+
+.sendbird-integrated-sample-app {
+  .integrated-app-theme--light {
+    background-color: var(--sendbird-light-background-50);
+  }
+  .integrated-app-theme--dark {
+    background-color: var(--sendbird-dark-background-600);
+  }
```

Comment:
Lets move the styles outside the scope of `. sendbird-integrated-sample-app`- try to do one level in SCSS